dcc-portal-server/src/main/java/org/icgc/dcc/portal/server/resource/entity/FileResource.java

```diff
@@ -162,15 +164,22 @@ public Long getUniqueDonorCount(
   @Path("/export")
   @Produces({TEXT_TSV, APPLICATION_JSON})
   @ApiOperation(value = "Exports repository file listings to a TSV or a JSON file.", response = File.class)
-  public Response getExport(
+  @SneakyThrows
+  public Response getExport (
       @ApiParam(value = API_FILTER_VALUE) @QueryParam(API_FILTER_PARAM) @DefaultValue(DEFAULT_FILTERS) FiltersParam filtersParam,
       @ApiParam(value = API_TYPE_VALUE) @QueryParam(API_TYPE_PARAM) @DefaultValue(DEFAULT_TYPE) String type) {
 
     final StreamingOutput outputGenerator =
         outputStream -> fileService.exportFiles(outputStream, query(filtersParam), type);
 
     // Make this similar to client-side export naming format
-    val fileName = String.format("repository_%s.%s", (new SimpleDateFormat("yyyy_MM_dd").format(new Date())), type);
+    val formatter = new SimpleDateFormat("MMM dd yyyy HH:mm:ss.SSS zzz");
+    long epochTime = 0L;
+    val currentTime = formatter.format(new Date());
+    val currentDate = formatter.parse(currentTime);
```

Comment:
There is actually some new handy date/time apis in Java 8 that we can use to make this simpler. 

```java
val epochTime = Instant.now().getEpochSecond();
val fileName = String.format("repository_%s.%s", epochTime, type);
```